.github/autobuild/windows.ps1

```diff
@@ -100,20 +132,46 @@ Function Ensure-JACK
         return
     }
 
-    echo "Install JACK2 64-bit..."
-    # Install JACK2 64-bit
-    choco install --no-progress -y jack --version "${JackVersion}"
+    # Set installer parameters for silent install
+
+    $JACKInstallParms = "/VERYSILENT", "/SUPPRESSMSGBOXES", "/NORESTART"
+
+    # Create cache directory if it doesn't exist yet
+
+    if (-not(Test-Path -Path "$DownloadCacheDir"))
+    {
+        New-Item -Path $DownloadCacheDir -ItemType "directory"
+    }
+
+    echo "Downloading 64 Bit and 32 Bit JACK installer (if needed)..."
+
+    Download-Dependency -Uri $Jack64Url -Name "JACK64.exe"
+    Download-Dependency -Uri $Jack32Url -Name "JACK32.exe"
+
+    # Install JACK 64 Bit silently via installer
+
+    echo "Installing JACK2 64-bit..."
+
+    $JACKInstallPath = "${DownloadCacheDir}\JACK64.exe"
+
+    & $JACKInstallPath $JACKInstallParms
+
     if ( !$? )
     {
-        throw "64bit jack installation failed with exit code $LastExitCode"
+        throw "64bit JACK installer failed with exit code $LastExitCode"
     }
 
-    echo "Install JACK2 32-bit..."
-    # Install JACK2 32-bit (need to force choco install as it detects 64 bits as installed)
-    choco install --no-progress -y -f --forcex86 jack --version "${JackVersion}"
+    echo "Installing JACK2 32-bit..."
+
+    # Install JACK 32 Bit silently via installer
+
+    $JACKInstallPath = "${DownloadCacheDir}\JACK32.exe"
+
+    & $JACKInstallPath $JACKInstallParms
+
     if ( !$? )
     {
-        throw "32bit jack installation failed with exit code $LastExitCode"
+        throw "32bit JACK installer failed with exit code $LastExitCode"
     }
```

Comment:
In the action log there is only a message that JACK2 is being installed. Maybe an else statement can be added here to report installation was completed? As this install is very silent, there is no other message in the log.